substrate/client/transaction-pool/src/graph/validated_pool.rs

```diff
@@ -671,7 +671,7 @@ impl<B: ChainApi> ValidatedPool<B> {
 	/// Consumers of this stream should use the `ready` method to actually get the
 	/// pending transactions in the right order.
 	pub fn import_notification_stream(&self) -> EventStream<ExtrinsicHash<B>> {
-		const CHANNEL_BUFFER_SIZE: usize = 1024;
+		const CHANNEL_BUFFER_SIZE: usize = 8192;
```

Comment:
The same question here, is it safe to merge it upstream? @michalkucharczyk?

Reply:
We can increase it, but it is not a source of a problem. Increasing this const will not solve anything.

The real problems is here:
https://github.com/paritytech/polkadot-sdk/blame/3ce8eadfda65ed4df846a4bbc08b9cca6a9f7cb0/substrate/client/network/src/protocol/notifications/handler.rs#L101

We are now investigating it a bit deeper, will share ticket in this thread soon.

Reply:
lets revert as @michalkucharczyk suggests
```suggestion
		const CHANNEL_BUFFER_SIZE: usize = 1024;
```